### DISCUSSION TITLE
Database option not available in "Data transfer target type and format" step of Export Data feature

### DISCUSSION THREAD

**Discussion by mkrigba** [dbeaver/dbeaver#20012] [Q&A] [answered]
### Description

I'm unable to choose "Database" as target when attempting to export data from a table in one database to another database.

According to Step 2 in https://github.com/dbeaver/dbeaver/wiki/Data-migration, Database should be an option but I don't see it in my version.
![image](https://github.com/dbeaver/dbeaver/assets/16820461/ef051649-c50c-458c-97e3-b70c27a301d9)

### DBeaver Version

DBeaver Lite Edition Enterprise Database Manager - Lite Edition Version 23.0.0.202303161222

### Operating System

Windows 10 Education Version 21H2

### Database and driver

We use several databases and drivers all of which exhibit the same lack of a Database option in the export feature.

### Steps to reproduce

1. Right click on any table in the Database Navigator
2. Select Export Data
3. Note that Database is not available in the Export target step

**Comment by E1izabeth** [comment, marked as answer]
DBeaver Light edition doesn't include Data Transfer to other databases feature. Please use DBeaver Community, Enterprise or Ultimate edition.

**Reply by mkrigba** [reply]
That's interesting that the free Community edition includes this feature, but it is excluded from Lite version.  That seems counterintuitive that I get less functionality in a paid version than I do a free version.  Can you revisit this decision to exclude this feature from the Lite edition?

**Reply by mkrigba** [reply]
Is there a table comparing features in Community to Lite similar to the table on the bottom of https://dbeaver.com/edition/?  Unfortunately, that table doesn't highlight the difference that I'm seeing either as there are green checkmarks on Data export/import line in all editions.

![image](https://github.com/dbeaver/dbeaver/assets/16820461/db5f5e03-bdac-46c3-8687-e0034cda771a)

If no table exists, can you share from your experience, features that exist in Lite that don't exist in Community and vice versa?

**Reply by E1izabeth** [reply]
Light Edition is intended for people, who wants to see less technical details while working with databases and supposed to not do any data migrations ever. Also it's because Light edition is the most lightweight edition for people, who wants no more than basic functionality.

**Reply by E1izabeth** [reply]
Please, contact our technical support team on the technical support system for licensed users https://dbeaver.com/submit-ticket/ or write to email [sales@dbeaver.com](mailto:sales@dbeaver.com) or [support@dbeaver.com](mailto:support@dbeaver.com). They will be glad to tell you about the differences in lots of details.
